plugins/backend/feedbin/feedbinAPI.vala

```diff
@@ -39,10 +39,18 @@ public class FeedbinAPI : Object {
 		if(user_agent != null)
 			m_session.user_agent = user_agent;
 
-		m_session.authenticate.connect((msg, auth, retrying) => {
-			if(!retrying)
-				auth.authenticate(this.username, this.password);
-		});
+		m_session.authenticate.connect(authenticate);
+	}
+
+	~FeedbinAPI()
```

Comment:
What does this line do? Should be removed i guess

Reply:
I'm not sure which line you mean, but the destructor is disconnecting the `authenticate` callback to prevent issues where the callback is called after the object is destroyed (I'm not sure if it's really necessary in this case, but it's safer to do it than not).